### ISSUE TITLE
Documentation about MISP update procedure

### ISSUE THREAD

**Issue by dpreuveneers** [MISP/MISP#6169]
**Is your feature request related to a problem? Please describe.**
The documentation in `https://github.com/MISP/MISP/blob/2.4/docs/UPDATE.md` states that it is very strongly recommended to upgrade MISP via the web interface. This will update MISP to the latest git commit. However, I am looking for an option to update MISP to the latest point release. Option 2 in the documentation offers a way to update to the latest point release, but this may break the update procedure in the MISP web interface. 

Also, the documentation of Option 2 does not consider the ownership of the files in the way that Option 1 does.

**Describe the solution you'd like**
Update the documentation of Option 2 to prepend the git commands with a 'sudo -u www-data' as in Option 1:
```
cd /var/www/MISP
sudo -u www-data git fetch
sudo -u www-data git checkout tags/$(git describe --tags `git rev-list --tags --max-count=1`)
sudo -u www-data git submodule update --init --recursive
```
However, doing this at the command line will still give a warning in the web interface that MISP is not on a branch. Would the following be an alternative to update MISP to the latest point release? 
```
cd /var/www/MISP
sudo -u www-data git fetch
sudo -u www-data git merge `git rev-list --tags --max-count=1`
sudo -u www-data git submodule update --init --recursive
```
It seems to the job without breaking the "Update MISP" procedure in the web interface.

**Comment by iglocska** [comment]
Indeed, we should label it correctly. There are two main strategies for updating your MISP:

1. Fetch the latest tagged version and stay on that release until you decide to update
````
cd /var/www/MISP
sudo -u www-data git fetch
sudo -u www-data git checkout tags/$(git describe --tags `git rev-list --tags --max-count=1`)
sudo -u www-data git submodule update --init --recursive
````

Does exactly that.

If you want to stay on the bleeding edge:

````
cd /var/www/MISP
git pull origin 2.4
sudo -u www-data git submodule update --init --recursive
````

will do the trick.
